dune-project

```diff
@@ -1,4 +1,4 @@
-(lang dune 3.7)
+(lang dune 3.16)
```

Comment:
I think this should raise the minimal version in `odoc.opam` file!

Reply:
Reverted